apsuite/commisslib/meas_ac_orm.py

```diff
@@ -0,0 +1,646 @@
+            data['cv_offsets'].append(cm.cycle_offset)
+            data['cv_kick_amplitudes'].append(conv(cm.cycle_ampl))
+            data['cv_kick_offsets'].append(conv(cm.cycle_offset))
+            data['cv_frequency'].append(cm.cycle_freq)
+            data['cv_num_cycles'].append(cm.cycle_num_cycles)
+            data['cv_cycle_type'].append(cm.cycle_type_str)
+
+        data['corrs_trig_delay_raw'] = self.devices['trigcorrs'].delay_raw
+        return data
+
+    def get_orbit(self):
+        """."""
+        orbx, orby = [], []
+        for bpm_name in self.sofb_data.bpm_names:
+            bpm = self.devices[bpm_name]
+            orbx.append(bpm.mt_posx)
+            orby.append(bpm.mt_posy)
+        return _np.array(orbx).T, _np.array(orby).T
+
+    def _create_devices(self):
+        # Create objects to convert kicks to current
+        self.devices.update({
+            n+':StrengthConv': StrengthConv(n, 'Ref-Mon')
+            for n in self.sofb_data.ch_names})
+        self.devices.update({
+            n+':StrengthConv': StrengthConv(n, 'Ref-Mon')
+            for n in self.sofb_data.cv_names})
+
+        # Create objects to interact with correctors
+        self.devices.update({
+            nme: PowerSupply(nme) for nme in self.sofb_data.ch_names})
+        self.devices.update({
+            nme: PowerSupply(nme) for nme in self.sofb_data.cv_names})
+
+        # Create object to get stored current
+        self.devices['currinfo'] = CurrInfoSI()
+
+        # Create BPMs trigger:
+        self.devices['trigbpms'] = Trigger('SI-Fam:TI-BPM')
+
+        # Create Correctors Trigger:
+        self.devices['trigcorrs'] = Trigger('SI-Glob:TI-Mags-Corrs')
+
+        # Create event to start data acquisition sinchronously:
+        self.devices['evt_study'] = Event('Study')
+        self.devices['evg'] = EVG()
+
+        # Create RF generator object
+        self.devices['rfgen'] = RFGen()
+
+        # Create BPMs
+        self.bpms = dict()
+        name = self.sofb_data.bpm_names[0]
+        self.bpms[name] = BPM(name)
+        self.csbpm = self.bpms[name].csdata
+
+        propties_not_interested = self.bpms[name].auto_monitor_status
+        propties_to_keep = ['GEN_XArrayData', 'GEN_YArrayData']
+        for ppt in propties_to_keep:
+            propties_not_interested.pop(ppt)
+
+        for i, name in enumerate(self.sofb_data.bpm_names):
+            if not i:
+                bpm = self.bpms[name]
+            else:
+                bpm = BPM(name)
+                self.bpms[name] = bpm
+            for propty in propties_not_interested:
+                bpm.set_auto_monitor(propty, False)
+            for propty in propties_to_keep:
+                bpm.set_auto_monitor(propty, True)
+                pvo = bpm.pv_object(propty)
+                self._flags[pvo.pvname] = _Flag()
+                pvo.add_callback(self._set_flag)
+        self.devices.update(self.bpms)
+
+    def _do_measure(self):
```

Comment:
maybe add the measurement RF column here?

Reply:
Right! I'll try to do it today, thanks!

Reply:
done!